pages/activities.jsx

```diff
@@ -131,8 +132,8 @@ var ActivitiesPage = React.createClass({
               src2x="/img/activities-page/2x/img-activity-01.png"
               title="Lo-Fi, No-Fi (Teaching Kit)"
               level="Beginner"
-              link="https://keyboardkat.makes.org/thimble/LTIxMDA3NTY0ODA=/lofi-nofi-teaching-kit"
-              developedBy="Kat Braybrooke, Karen Smith, Julia Vallera, Jess Klein, Chad Sansing"
+              link=" https://laura.makes.org/thimble/MTUyODMwNDY0/lofi-nofi-teaching-kit"
```

Comment:
Hmm what's up with the whitespace between the `"` and the `https`? Fix that and this is good to merge!